src/intrinsics/ecma262/Error.js

```diff
@@ -150,6 +141,15 @@ export function build(name: string, realm: Realm, inheritError?: boolean = true)
       message.throwIfNotConcrete();
     }
 
+    // Build a text description of the stack.
+    let stackDesc = {
+      value: buildStack(realm, O),
```

Comment:
buildStack does not yet handle abstract messages. You can add such support by emitting the stack as a string template. Alternatively, you could just comment out the throw with a TODO comment, along with an issue describing what needs to be done.

Reply:
:+1: